mithril.js

```diff
@@ -218,9 +218,34 @@
 		return list
 	}
 
+	var applyIEPerf = (
+			typeof document !== "undefined" &&
+			typeof document.documentMode === "number"
+		) ||
+		(
+			typeof navigator !== "undefined" &&
+			typeof navigator.userAgent === "string" &&
+			/\bEdge\/\d/.test(navigator.userAgent)
+		)
+
 	function insertNode(parentElement, node, index) {
-		parentElement.insertBefore(node,
-			parentElement.childNodes[index] || null)
+		if (applyIEPerf) {
+			if (!parentElement._children) {
+				parentElement._children = []
```

Comment:
You do have a good point. I've been busy lately working on other things unrelated to Mithril, so my mind isn't quite as fresh with dealing with Mithril's internals as it was before.